### PR TITLE
Add Contact form with validations and submitted data to Formspree

### PR DESCRIPTION
**Pull request by Metaverse-Nabeel**
### Following changes are made in this PR.

1. Contact Form section is added to the index.html file
2. Validations are added for the input and text area as per requirements
3. Submitted the form data at Formspree.io
4. Checked for Linter errors and corrected them
5. Updated README.md to correct the LICENSE.md link
6. Updated the design to make it pixel perfect as per the Figma design

@MarcoDDM: Made a commit by changing the form's width and alignment of the submit button.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -292,5 +292,46 @@ <h4 class="titleSM">LET’S CONNECT</h4>
         </ul>
       </div>
     </section>
+    <section class="contactForm">
+      <div class="contactBg">
+        <div class="contactTop">
+          <h3 class="contactTitle">Contact me</h3>
```
Comment (amejid) [inline index.html:298]: - [ ] Great job integrating the form with Formspree service. :+1: Let's work on minor details to match the design. Kindly increase the space above this title `Contact me`. Please also adjust the space between the paragraph and the text input. ✍️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||

**Review by amejid — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter checks passing :heavy_check_mark: 
- Good UI design :heavy_check_mark: 
- Formspree service integrated successfully :heavy_check_mark: 
- HTML validations are working :heavy_check_mark: 

## Required Changes ♻️

- [x] When clicking the License link in the README file it is giving `404` error. I think it is happening because the link is referencing `LICENSE` while the file is named `LICENSE.md`. Kindly rename either of them to fix the issue. 😎️

Screenshot from 2023-02-21 21-28-31



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Metaverse-Nabeel) [reply index.html:298]: Hi @amejid 

Thank you for the detailed review. I shall make these changes and make a new request.

**Review by osamaashraf6 — APPROVED** [review]
Hi @Metaverse-Nabeel 👋 , 
# STATUS: APPROVED! 🥇🟢🟢💯 


Your project is complete!  There is nothing else to say other than... it's time to merge it : ship it: 
Congratulations! 🎉 

## Highlights💯🎈🪄✔️

- [x] Good adding the validation of the project ✔️
- [x] Good using the Form Spree ✔️
- [x] Descriptive READme file in a professional way.✔️
- [x] No linter errors. ✔️
- [x] Good usage in correcting GitHub flow.✔️
- [x] Nice design.✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**